person_capture/gui_app.py

```
@@ -618,14 +618,34 @@ def _span_index_for(self, f, spans):
 
     def _prescan_skip_forward(self, cap, count: int) -> int:
         """Discard up to ``count`` decoded frames without restarting the reader."""
+
+        def _at_known_eof() -> bool:
+            try:
+                total = int(cap.get(cv2.CAP_PROP_FRAME_COUNT) or 0)
+            except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)